### ISSUE TITLE
PYTHONPATH in tests

### ISSUE THREAD

**Issue by johnomotani** [boutproject/BOUT-dev#695]
PYTHONPATH has to be set correctly for make check-integrated-tests to work. I can imagine a potential for confusion if someone has multiple copies of BOUT on the same system, and tries to test one while PYTHONPATH points to the tools/pylib of a different (e.g. older) version. Is there a way to override it in the makefile so that the version of tools/pylib belonging to the copy of BOUT being tested is used, instead of whatever is in the user's PYTHONPATH, or at least print some sort of warning if PYTHONPATH doesn't point to the copy of BOUT being tested?

**Comment by d7919** [comment]
This should be possible, perhaps changing

~~~~
        @cd tests/integrated; ./test_suite_make && ./test_suite
~~~~

to

~~~~
        @cd tests/integrated; ./test_suite_make && PYTHONPATH=${PWD}/../../tools/pylib/:${PYTHONPATH} ./test_suite
~~~~

(although this could be tidier).

**Comment by dschwoerer** [comment]
solved by #713
